src/System.Reflection.DispatchProxy/src/System.Reflection.DispatchProxy.csproj

```
@@ -0,0 +1,15 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
```

nit: readability, add empty line after Project tag line:

```suggestion

  <PropertyGroup>
```